nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js

```diff
@@ -412,9 +412,9 @@
 
                 // create object whose keys are the parameter name and the values are the parameter values
                 var paramsObject = {};
-                params.forEach(function (v, k) {
-                    paramsObject[k] = v;
-                });
+                for(var pair of params.entries()) {
```

Comment:
Good catch! As it turns out we dont need to loop over the URLSearchParams...we can just use the .toString(). I have updated this PR to reflect this change. Please test and let me know if it works for you.